specification/synapse/resource-manager/Microsoft.Synapse/preview/2019-06-01-preview/workspace.json

```diff
@@ -668,6 +668,50 @@
         }
       }
     },
+    "EncryptionDetails": {
```

Comment:
Can these encryption details be modified to be closer to the common contract defined here:  https://github.com/Azure/azure-resource-manager-rpc/blob/master/v1.0/common-api-contracts.md#customer-managed-key-encryption

Reply:
We've already moved forward with this design. Is this a blocking requirement?

Reply:
CMK should be a common pattern as much as possible so users don't have to re-learn the API characteristics for each resource that support CMK.  API reviews should also take place before the API is implemented for exactly this reason

Reply:
Ok I was not told that this was the pattern by team members in my team. In fact, we prioritized the work related to the CMK feature to unblock partner teams. It's too late to make this kind of change at this point.

Reply:
CMK is a high pri feature for all teams.  However, it is also important that it is as consistent as possible for our end users.  Looking at this again its close enough given the caveats of your needs